### DISCUSSION TITLE
0.4.1g

### DISCUSSION THREAD

**Discussion by alienatedsec** [alienatedsec/yi-hack-v5#340] [Announcements]
## Breaking changes !!!
* There is a new baseline `0.4.1g`, so you need to prepare your SD card for it - follow https://github.com/alienatedsec/yi-hack-v5/discussions/267
* The camera web page is no longer under port `8080` but it is back to default port `80` - access your camera web page using its IP only. Those with proxy environments or `Home Assistant` integrations need to amend their settings.
* The new baseline is needed to fix OTA upgrades #326 - cameras on `0.4.1e` and `0.4.1f` can upgrade from the WebUi if the SD card is still FAT32 - future non-baseline upgrades will be released without this requirement. 
* We need testers, so if you have no physical access to your camera - don't upgrade, as it may require fixing by downgrading etc.

## What's Changed
* Fix for `yi_home` and `yi_dome` - the `wpa_supplicant` upgrade in the baseline image was prevented by the YI upgrade of an older file
* Recovered lost `system.sh` upgrades and other functions
* Cleared the libs folder on the `home` partition - attempt to fix #312 and #291 

## What is still not working
* #336 - unable to find the right `sleep` settings as suggested by @roleoroleo - please test and suggest if possible - it does not work on my cameras whatever I try - `yi_home_1080p - 48US` and `yi_dome - 43US`
* #333 - Still investigating the MQTT status changes

**Full Changelog**: https://github.com/alienatedsec/yi-hack-v5/compare/0.4.1f...0.4.1g

<hr /><em>This discussion was created from the release <a href='https://github.com/alienatedsec/yi-hack-v5/releases/tag/0.4.1g'>0.4.1g</a>.</em>

**Comment by fraserp** [comment]
Installed and working - also the interface on my Yi outdoor seems way snappier/more responsive than before (not sure if that is due to intentional change)

**Reply by alienatedsec** [reply]
Please follow #312 as the boot looping is still the case for `v0.4.1g`
